src/services/api.ts

```diff
@@ -61,4 +61,105 @@ apiClient.interceptors.response.use(
   },
 );
 
+//connection api
```

Comment:
You want to create a separate services file, ie connectionService.ts and put all this logic in there. Use the apiClient object exported from this file to get the basic configuration for the axios client.